lib/changes/toggleList.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+import { Document, Node } from 'slate';
+import Options from '../options';
+import { wrapInList } from '.';
+
+const LIST_TYPES = ['ol_list', 'ul_list'];
+
+type MappedNode = {
+    depth: number,
+    node: Node
+};
+
+function isList(blockType: string) {
+    return LIST_TYPES.includes(blockType);
+}
+
+function isItem(blockType: string) {
+    return blockType === 'list_item';
+}
+
+function filterListDescendants(node: Node) {
+    return isList(node.type) || isItem(node.type);
+}
+
+function mapListDescendants(document: Document) {
+    return (node: Node) => ({
+        depth: document.getDepth(node.key),
+        node
+    });
+}
```

Comment:
it would be cooler with arrow functions, especially if you return a func, but the whole lib uses normal functions, so yeah, we should stick to them I guess 🙄